src/SwiftUI/Mac/NSHostingView.cs

```diff
@@ -12,13 +12,13 @@ public unsafe class NSHostingView : NSView, /*INSUserInterfaceValidations,*/ INS
 	{
 		public static NSHostingView Create (View view)
 		{
-			using (var handle = view.GetHandle ())
-				return Create (handle.Pointer, view.ViewType);
+			using (var handle = view.GetSwiftHandle ())
+				return Create (handle.Pointer, handle.SwiftType);
```

Comment:
nit: In this case, could we not just pass `handle`, that obtain the Pointer/SwiftType inside the `Create` call?

Reply:
Great idea! I also took the opportunity to switch to using regular constructors instead of `Create` static methods